scripts/globals/abilities/berserk.lua

```diff
@@ -14,7 +14,7 @@ function onAbilityCheck(player,target,ability)
 end
 
 function onUseAbility(player,target,ability)
-    player:addStatusEffect(dsp.effect.BERSERK,1,0,180)
+    player:addStatusEffect(dsp.effect.BERSERK,player:getMod(dsp.mod.BERSERK_EFFECT),0,180)
```

Comment:
Is there ever a time berserk potency is less than 25? would make more sense to set this 25+modifier than to repeatedly say 25 and the effect power in the effect script.

Reply:
Yes, potentially a separate value for the defense penalty if AF2 feet are equipped, but I think that can be handled entirely by latent since they need to be equipped for it to take effect and its lost on un-equip.